### PR TITLE
feat: add imagepullsecrets in helm chart

### PR DESCRIPTION
**Pull request by dza89**
Signed-off-by: daan <dhapotter@gmail.com>

Add imagepullsecrets to daemonset and deployment

### CODE REVIEW

**Comment by dza89** [comment]
I don't know what version to make? 0.12.0 or 1.0.1? 
Also, it's quite weird to put tgzs in your repository.

**Comment by dza89** [comment]
Alright, I'm not going to work on this MR anymore because the amount of work you have to do to add three lines is just too much. 
So either you guys can fix it up, otherwise close this mr

**Comment by andyzhangx** [comment]
no worry, I will fix it up

**Comment by dza89** [comment]
@andyzhangx 

Thanks, sorry for my rant.
Seems like alot of azure charts are based on old helm boilerplating, because this is default in the latest boilerplating.

File: charts/latest/blob-csi-driver/templates/csi-blob-node.yaml [hunk 1]
```diff
@@ -13,6 +13,10 @@ spec:
 {{ include "blob.labels" . | indent 6 }}
         app: csi-blob-node
     spec:
+      {{- if .Values.imagePullSecrets }}
```
Comment (andyzhangx) [inline charts/latest/blob-csi-driver/templates/csi-blob-node.yaml:16]: only need to make change under charts/latest, and then run `helm package charts/latest/blob-csi-driver -d charts/latest/` to update tgz file, thanks.

Comment (dza89) [reply charts/latest/blob-csi-driver/templates/csi-blob-node.yaml:16]: Done(?)

File: charts/latest/blob-csi-driver/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: latest
 description: Azure Blob Storage CSI driver
 name: blob-csi-driver
-version: v1.0.1
```
Comment (andyzhangx) [inline charts/latest/blob-csi-driver/Chart.yaml]: revert this, no need to change version here.

File: charts/latest/blob-csi-driver/values.yaml [hunk 3]
```diff
@@ -20,6 +20,9 @@ image:
     tag: v1.1.0
     pullPolicy: IfNotPresent
 
+imagePullSecrets: []
```
Comment (andyzhangx) [inline charts/latest/blob-csi-driver/values.yaml:23]: it's better write like this:
https://github.com/helm/charts/blob/master/stable/cert-manager/values.yaml#L5-L9

Comment (dza89) [reply charts/latest/blob-csi-driver/values.yaml:23]: i don't know what i'm looking at? It looks the same to me?
Are you referring to the comments?

Comment (andyzhangx) [reply charts/latest/blob-csi-driver/values.yaml:23]: yes, just comments, it's a little better

Comment (dza89) [reply charts/latest/blob-csi-driver/values.yaml:25]: Done

**Review by andyzhangx — APPROVED** [review]
/ok-to-test

**Pull request merged**